definitions/m5/m5.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: m5-release-pipeline
+  namespace: managed
+spec:
+  params:
+    - name: mapping-file
+      type: string
+      description: mapping to apply to snapshot
+  tasks:
+    - name: get-mapping-file
+      taskRef:
+        name: get-mapping-file
```

Comment:
This won't work as the Task will be defined in a bundle. I would actually define separate bundles for them.

Reply:
The entire directory becomes a bundle, yes. The problem is when it get's used. The Pipeline is fetched from the bundle but it will try to find the task in the cluster and not in the bundle. For that you should add a bundle line here pointing to the task in the bundle that you are about to create. Thus, better to separate them in two different bundles.

Reply:
I added a line below this after your comment. Is this what you mean? Also, why two different bundles? What goes in one and not the other for m5 (which is our only defined release pipeline for now)

Reply:
Yeah, ugly but this would work.

Reply:
Why ugly? What do you want me to add instead?

Reply:
It's ugly because you will have a bundle with a Pipeline targeting another tag of the same bundle. This is just a test pipeline. Go ahead and create a `release-m5` bundle or something like that with the actual definition of the Release Pipeline for M5 even if it contains just the call to this Task. You could call this bundle `component-mapping` or anything like that. We can discuss this offline if you prefer it.